tests/integration/test_install_markers.py

```diff
@@ -37,14 +36,7 @@ def test_platform_python_implementation_marker(PipenvInstance):
     incorrectly.
     """
     with PipenvInstance() as p:
-        with open(p.pipfile_path, 'w') as f:
-            contents = """
-[packages]
-depends-on-marked-package = "*"
-            """.strip()
-            f.write(contents)
-
-        c = p.pipenv('install')
+        c = p.pipenv('install depends-on-marked-package')
```

Comment:
This was problematic because this only exists in our local pypi, but that Pipfile doesn't have the sources -- not sure why this hadn't cropped up before now but I think it has to do with I ported the intended fix for the searching of all indexes to this PR: https://github.com/pypa/pipenv/pull/5220 as well since it started as a copy of `pip_install` which had this bug.